File: blog-post-2/programjava/Fibonacci.java
package programjava;

import java.util.Scanner;

public class Fibonacci {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.print("Enter number: ");
        int number = in.nextInt();
        in.close();
        int next = 1,prev = 0,temp;

        System.out.print("0 1 ");
        for(int i = 0; i < number - 2; i++){
            System.out.print((next + prev) + " ");
            temp = next + prev;
            prev = next;
            next = temp;
        }
    }
}

File: blog-post-2/c/Fibonacci.c
#include <stdio.h>

int main()
{
    int number,next = 1,prev = 0,temp;
    printf("Enter number: ");
    scanf("%d",&number);
    if(number == 1){ 
        printf("0 ");
    }
    else
    {
        printf("0 1 ");
    }
    
    for (int i = 0; i < number - 2; i++)
    {
        printf("%d ",(next + prev));
        temp = next + prev;
        prev = next;
        next = temp;
    }
}